community/values/src/test/java/org/neo4j/values/storable/ValueComparisonTest.java

```diff
@@ -87,7 +84,10 @@
             Float.MAX_VALUE,
             Double.MAX_VALUE,
             Double.POSITIVE_INFINITY,
-            Double.NaN
+            Double.NaN,
+
+            // OTHER
+            null
```

Comment:
Yes, `null` is last, https://github.com/neo4j/neo4j/blob/3.3/community/cypher/cypher-compiler-3.3/src/main/java/org/neo4j/cypher/internal/compiler/v3_3/common/CypherOrderability.java#L131